srcpkgs/sayonara-player/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'sayonara-player'
+pkgname=sayonara-player
+version=1.7.0
```

Comment:
The version should include the "stable3" component.

Reply:
xlint does not allow me to add `-stable3`
so should i just do `1.7.0stable3` ?